README.md

```diff
@@ -1,7 +1,7 @@
 <div align="center">
     <p><a href="#"><a href="https://ollama.com/"><img alt="ollama" src="./imgs/ollama-icon.png" width="200px" /></a><img alt="+" src="./imgs/grey-plus.png" width="100px" /></a><a href="https://www.minecraft.net/"><img alt="discord" src="./imgs/minecraft-icon.png" width="190px" /></a></p>
     <h1>Minecraft Ollama Integration</h1>
-    <h3><a href="#"></a>Ollama as your Minecraft World Overseer</h3>
+    <h3><a href="#"></a>Ollama as your Minecraft World AI Companion</h3>
```

Comment:
I remember speaking about this months ago, AI Companion has a nice (PC) ring to it.